src/graph/plugin_host.rs

```diff
@@ -4,6 +4,7 @@ use clack_host::events::io::EventBuffer;
 use clack_host::events::spaces::CoreEventSpace;
 use clack_host::events::{Event, EventFlags, EventHeader};
 use clack_host::utils::Cookie;
+use crossbeam_channel::Sender;
```

Comment:
What's the purpose of using crossbeam in the plugin host here?

Reply:
This isn't used in the plugin host itself, I'm only passing the `DSEngineEvent` sender to the `on_handle` function so it can send the `GuiClosed` events, regardless of the `OnIdleResult` it returns.